pkg/config/agent/config.go

```diff
@@ -292,6 +292,9 @@ type MulticlusterConfig struct {
 	// Enable Multi-cluster NetworkPolicy which allows Antrea-native policy ingress rules to select peers
 	// from all clusters in a ClusterSet.
 	EnableStretchedNetworkPolicy bool `yaml:"enableStretchedNetworkPolicy,omitempty"`
+	// Enable Multi-cluster Pod to Pod Connectivity which allows one Pod access another Pod in other member
+	// clusters directly. It also requires the Pod CIDRs in Multi-cluster configuration.
```

Comment:
"This feature also requires Pod CIDRs to be provided in the Multi-cluster Controller configuration"

Reply:
Done.